src/pudl/metadata/constants.py

```diff
@@ -259,6 +259,34 @@
         "government",
         "federal",
     ],
+    "msha": [
+        "msha",
+        "mshamines",
+        "mine safety and health administration",
+        "mines",
+        "mining",
+        "coal",
+        "metal",
+    ],
+    "phmsa": [
+        "phmsa",
+        "phmsagas",
+        "pipelines and hazardous materials safety administration",
+        "pipelines",
+        "natural gas",
+        "transmission",
+        "distribution",
+        "gathering",
+        "liquified natural gas",
+        "lng",
+        "underground natural gas storage",
+    ],
+    "eia_water": [
```

Comment:
"water energy nexus" and "energy water nexus" might also be common search terms for this kind of information.